csfieldguide/utils/check_heading_tree.py

```diff
@@ -0,0 +1,20 @@
+"""Check for any duplicate headings in the heading tree."""
+
+from utils.errors.DuplicateHeadingFoundInMarkdownFileError import DuplicateHeadingFoundInMarkdownFileError
+
+
+def check_heading_tree(heading_tree, md_file_path):
```

Comment:
Travis is failing because this and traverse_node are missing docstrings

Reply:
Thanks, this is now fixed.